WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -399,6 +397,12 @@ extension PushNotificationsManager {
     }
 }
 
+private extension Date {
+    var components: DateComponents {
```

Comment:
It would be nice to move this into WordPressShared (Date+Helpers) but we can do this separately :)

Reply:
I'll create a separate PR for this 😊